posts/2019/04/uarray-intro.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `uarray`: A Generic Override Framework for Methods\n",
+    "\n",
+    "`uarray` is an override framework for methods in Python. In the scientific Python ecosystem, and in other similar places, there has been one recurring problem: That similar tools to do a job have existed, but don't conform to a single, well-defined API. `uarray` tries to solve this problem in general, but also for the scientific Python ecosystem in particular, by defining APIs independent of their implementations.\n",
+    "\n",
+    "## The Scientific Python Ecosystem: How Fragmentation Emerged\n",
```

Comment:
I'm not sure about the logic of the first and second paragraph. There were about 10 years in between NumPy unifying everything and then new array libraries appearing. So this looks a little odd. Maybe change this section title to something like `Array libraries in the scientific Python ecosystem`?